src/containers/AddressPage/index.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { connect } from "react-redux";
+import { routes } from "../Router";
+import { push } from "connected-react-router";
+import { signUpAddress } from "../../actions/auth"
+import { getProfile } from "../../actions/"
+import Header from '../../components/Header';
+import MainButtonComponent from '../../components/MainButton'
+import { ContainerAddressPage, TextRegister, InputAddress, InputNumber, InputHouse, InputNeighborhood, InputCity, InputState} from './styled';
+
+class AddressPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            street: "",
+            number: "",
+            neighbourhood: "",
+            city: "",
+            state: "",
+            complement: ""
+        }
+    }
+
+    componentDidMount(){
+        this.props.getProfile()
+    }
+
+    handleSubmit = (event) => {
+        event.preventDefault()
+
+        const { street, number, neighbourhood, city, state, complement } = this.state
+
+        this.props.createdAddress(street, number, neighbourhood, city, state, complement, 'address')
+    }
+
+    clearNewTaskValue = () => {
+        this.setState({ name: "", email: "", cpf: "", password: "", passwordConfirm: ""});
+    };
+
+    handleFieldChangeStreet = (event) => {
+        this.setState({ street: event.target.value })
+    }
+
+    handleFieldChangeNumber = (event) => {
+        this.setState({ number: event.target.value })
+    }
+
+    handleFieldChangeNeighbourhood = (event) => {
+        this.setState({ neighbourhood: event.target.value })
+    }
+
+    handleFieldChangeCity = (event) => {
+        this.setState({ city: event.target.value })
+    }
+
+    handleFieldChangeState = (event) => {
+        this.setState({ state: event.target.value })
+    }
+
+    handleFieldChangeComplement = (event) => {
+        this.setState({ complement: event.target.value })
+    }
+
+    render() {
+
+        const { street, number, neighbourhood, city, state, complement } = this.state;
+
+        return(
+
+            <div>
+                <Header title={''} isArrowBackVisible={true}/>
```

Comment:
Se for pra deixar o `title` vazio, é preferível que vocês omitam ele de vez.

Outro toque é que se a prop é booleana, a simples presença dela é interpretada como `true` e a ausência dela é interpretada como `false`, ou seja, o exemplo abaixo faz a mesma coisa que o código de vocês com menos ruído.

```jsx
<Header isArrowBackVisible />
```